db/migrations/20230327081950_create_single_volume_pages_table.sql

```diff
@@ -0,0 +1,12 @@
+-- migrate:up
+CREATE TABLE single_volume_pages (
+  id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+  single_volume_page_id INT NOT NULL,
+  image_url varchar(200) NOT NULL,
```

Comment:
URL은 길이가 길어질 수 있기 때문에 2000정도로 넉넉히 주면 좋을 것 같아요.
```suggestion
  image_url varchar(2000) NOT NULL,
```